target/linux/ramips/dts/mt7620a_tplink_archer-mr200.dts

```diff
@@ -168,9 +168,8 @@
 };
 
 &ethernet {
-		mtd-mac-address = <&rom 0xf100>;
-		mediatek,portmap = "llll";
```

Comment:
What about using "llllw" here, as the only valid existing configuration?

Reply:
It does not have WAN port, so I don't feel that's really needed.